modules/SerialCommHub/tiny_modbus_rtu.cpp

```diff
@@ -263,18 +269,19 @@ bool TinyModbusRTU::open_device(const std::string& device, int _baud, bool _igno
 
 int TinyModbusRTU::read_reply(uint8_t* rxbuf, int rxbuf_len) {
     struct timeval timeout;
-    timeout.tv_sec = 0;
-    timeout.tv_usec = MODBUS_RX_INITIAL_TIMEOUT_MS * 1000; // 500msec intial timeout until device responds
+    timeout.tv_sec = 6;
```

Comment:
Why an additional 6 seconds?

Reply:
We run into timeouts when waiting for larger datablobs over the modbus... We may do this as a parameter

Reply:
Okay - while going from 500 to 6500 ms seems extreme.
Please adapt the misleading comment one line below my remark.

Reply:
I would also be in favor of making this a config option with a default less then the 6 seconds. This will create problems on some systems if the whole bus hangs that long.

Reply:
+1 for making this configurable. It is a Qwello problem that we need 6 seconds.